ape_hardhat/providers.py

```diff
@@ -373,6 +373,19 @@ def get_transaction_trace(self, txn_hash: str) -> Iterator[TraceFrame]:
         for frame in frames:
             yield TraceFrame(**frame)
 
+    def get_call_tree(self, txn_hash: str) -> CallTreeNode:
+        receipt = self.get_transaction(txn_hash)
+        root_node_kwargs = {
+            "address": receipt.receiver,
+            "calldata": receipt.input_data,
+            "call_type": CallType.MUTABLE,
+            "gas_cost": receipt.gas_used,
+            "gas_limit": receipt.gas_limit,
+            "value": receipt.value,
+        }
+        trace = self.get_transaction_trace(txn_hash)
+        return get_calltree_from_trace(trace, **root_node_kwargs)
```

Comment:
i renamed to `get_calltree_from_geth_trace` here
https://github.com/ApeWorX/evm-trace/pull/7/commits/de7d0f868e61befb85b0274a6f41abfde6ed503e